multiuser/integration-tests/che-multiuser-mysql-tck/pom.xml

```diff
@@ -252,7 +252,11 @@
                                             <port>jdbc.port:3306</port>
                                         </ports>
                                         <wait>
-                                            <log>ready for connections</log>
```

Comment:
Do we really need to remove `log` here?

Reply:
Yes, this was the reason the tck was getting stuck (at least on my machine). The new checking based on the "connectability" of the port seems to be working better.

Reply:
I see. I thought that it was printing log but not checking. Thanks